src/test/java/uk/gov/justice/probation/courtcaseservice/controller/CourtCaseControllerIntTest.java

```diff
@@ -508,6 +507,176 @@ void shouldReturnNotFoundForDeletedCase() {
 
     }
 
+    @Nested
+    class GetCaseByHearingIdAndDefendantId {
+        private static final String PATH = "/hearing/{hearingId}/defendant/{defendantId}";
+        private static final String CASE_ID = "1f93aa0a-7e46-4885-a1cb-f25a4be33a00";
+        private static final String HEARING_ID = "1f93aa0a-7e46-4885-a1cb-f25a4be33a00";
+
+        @Test
+        void givenKnownCaseIdWithNoOffender_whenGetCase_thenReturn() {
+
+            var response = given()
+                .given()
+                .auth()
+                .oauth2(getToken())
+                .when()
+                .header("Accept", "application/json")
+                .get(PATH, HEARING_ID, DEFENDANT_ID)
+                .then()
+                .statusCode(200);
+
+            response
+                .body("caseId", equalTo(CASE_ID))
+                .body("hearingId", equalTo(HEARING_ID))
+                .body("offences", hasSize(2))
+                .body("offences[0].offenceTitle", equalTo("Theft from a shop"))
+                .body("offences[0].offenceSummary", equalTo("On 01/01/2015 at own, stole article, to the value of £987.00, belonging to person."))
+                .body("offences[0].act", equalTo("Contrary to section 1(1) and 7 of the Theft Act 1968."))
+                .body("offences[1].offenceTitle", equalTo("Theft from a different shop"))
+                .body("probationStatus", equalTo("Current"))
+                .body("probationStatusActual", equalTo("CURRENT"))
+                .body("previouslyKnownTerminationDate", equalTo(LocalDate.of(2010, Month.JANUARY, 1).format(DateTimeFormatter.ISO_LOCAL_DATE)))
```

Comment:
This is an Offender field but the test name is `givenKnownCaseIdWith*NoOffender*_whenGetCase_thenReturn`, needs renaming or maybe it should be fetching a different defendant?